compiler/AST/expr.cpp

```diff
@@ -3786,7 +3786,8 @@ void CallExpr::prettyPrint(std::ostream *o) {
         argList.last()->prettyPrint(o);
         *o << ")";
       } else if (strcmp(expr->unresolved, "chpl_build_bounded_range") == 0 ||
-                 strcmp(expr->unresolved, "chpl_build_partially_bounded_range") == 0 ||
+                 strcmp(expr->unresolved, "chpl_build_low_bounded_range") == 0 ||
+                 strcmp(expr->unresolved, "chpl_build_high_bounded_range") == 0 ||
                  strcmp(expr->unresolved, "chpl_build_unbounded_range") == 0) {
         argList.first()->prettyPrint(o);
         *o << "..";
```

Comment:
After some digging, I don't think this code is used anymore. I'm going to put an `INT_ASSERT(false);` in here for now. I would just remove it, but I don't think we'd ever notice if it did get used again. I think it used to be used, but the chpldoc stuff that used to use it now calls into AstToText. I believe this function is only used by chpldoc for printing out the return signature (but obviously you can't specify a range literal as a return type, so I think it'd dead code now .)

Man.. I used the word used a lot in that paragraph..

There is some other code that needs to be updated. https://github.com/chapel-lang/chapel/blob/master/compiler/AST/AstToText.cpp#L999

It doesn't work for anything but fully bounded ranges (not a new bug though) 

I added a future for this with https://github.com/chapel-lang/chapel/pull/3167